### PR TITLE
Adjust spelling for countries using British English

### PR DESCRIPTION
**Pull request by pbstFusion**
I have are a very particular set of skills. Skills I have acquired over a very long career. Skills that make me a nightmare for people like you. If you merge the request, that'll be the end of it. I will not look for you, I will not pursue you, but if you don't, I will look for you, I will find you and I will forcibly teach you how to install Fortnite in binary.

### CODE REVIEW

**Comment by pbstFusion** [comment]
I can already feel the angry comments,

**Review by GalacticInspired — CHANGES_REQUESTED** [review]
I request changes.

**Comment by pbstFusion** [comment]
> You're all idiots, discussing over the arrangement of a single letter, over american and british english. American english is clearly superior. I don't care if you come from Worse Britain or Kan'ah'da.

If it doesn't matter, why are you arguing about it? :)

File: MainModule/Client/UI/Default/CommsCenter.lua [hunk 1]
```diff
@@ -103,7 +103,7 @@ return function(data)
 	
 	local reCountries = {"GB", "CA", "AU", "IE", "NZ", "ZA", "GS"}
 		local success, isBritishSpelling = xpcall(function()
-			if table.find(reCountries, service.LocalizationService:GetCountryRegionForPlayerAsync(service.Players.LocalPlayer)) => 0 then
+			if table.find(reCountries, service.LocalizationService:GetCountryRegionForPlayerAsync(service.Players.LocalPlayer)) => 1 then
```
Comment (GalacticInspired) [inline MainModule/Client/UI/Default/CommsCenter.lua:106]: Disgusting

**Review by GalacticInspired — CHANGES_REQUESTED** [review]

**Comment by Awesomewebm** [comment]
This is one of the most unnecessary PRs ever :D

File: MainModule/Client/UI/Default/CommsCenter.lua (hunk 1 above)
Comment (pbstFusion) [reply MainModule/Client/UI/Default/CommsCenter.lua:106]: didn't ask

**Comment by pbstFusion** [comment]
> This is one of the most unnecessary PRs ever :D
No, that would be the one that randomly removed the functionality in the first place

**Review by joritochip — APPROVED** [review]

**Comment by GalacticInspired** [comment]
This is stupid

**Comment by pbstFusion** [comment]
> This is stupid

You are entitled to your own opinion, even if it is false/incorrect/wrong.

**Comment by GalacticInspired** [comment]
it's not useful in anyway, and it wouldn't make sense that this is the only british spelled word in whole adonis

**Comment by ccuser44** [comment]
Oi mate do you have a loisence for that?

**Comment by ccuser44** [comment]
But fr do we reslly need a single letter thing here? Lol. If so then maybe we should translate all of Adonis. If not then this is unnecessary

**Comment by GalacticInspired** [comment]
it’s either all adonis or nothing

why should this command be the only thing to stand out?

**Comment by ccuser44** [comment]
also this code would even work because `table.find` can return `nil`

**Comment by ccuser44** [comment]
Wait the weirdest thing I realised is, that this command had a version before this where it chsnged it based on american english

Why was thid implemented in  the first place?

**Comment by GalacticInspired** [comment]
Sceleratis wants to keep adonis AE (American English) based, so he just
changed all cent*re *to center.

**Comment by ccuser44** [comment]
> Sceleratis wants to keep adonis AE (American English) based, so he just
> changed all cent*re *to center.

Or we could just do 
`math.random(1, 2) ==2 and "Centre" or "Center"`

**Comment by GalacticInspired** [comment]
Die

**Comment by Sceleratis** [comment]
On a more realistic note, if we are going to do this we would need to do this via localization, not one off/one-by-one basis... otherwise we're going to have word checks everywhere adding additional overhead and code that doesn't need to be there.

**Comment by GalacticInspired** [comment]
Just close this

**Comment by moo1210** [comment]
Doesn't seem so. Only see en-us in CoreScriptLocalization.

**Comment by LolloDev5123** [comment]
> Doesn't seem so. Only see en-us in CoreScriptLocalization.

pbstFusion go mine crypto to my address
`3P44apMLSALiV8yJDJdZNpmpGC1aWiowXw`

**Comment by GalacticInspired** [comment]
image

**Comment by pbstFusion** [comment]
That is the pinnacle of satire.

**Comment by pbstFusion** [comment]
Actually I have just has an infinite iq idea

**Pull request closed without merging**